### PR TITLE
Initial readme

### PR DESCRIPTION
**Pull request by yi2020**

### CODE REVIEW

**Comment by alonashkenazi** [comment]
@yi2020 can you use the branch to include the ticket number? I have started a branch for this task you can see this PR: https://github.com/indeni/dragoneye/pull/8/files

File: README.md [hunk 1]
```diff
@@ -21,34 +21,35 @@ pip install .
 
 ## Programmatic Usage
 Create an instance of one of the CollectRequest classes, such as AwsAccessKeyCollectRequest, AwsAssumeRoleCollectRequest, AzureCollectRequest and call the `collect` function. For example:
-```
+```python
 from dragoneye import AwsScanner, AwsCloudScanSettings, AzureScanner, AzureCloudScanSettings, AwsSessionFactory, AzureAuthorizer
 
-
+### AWS ###
```
Comment (alonashkenazi) [inline README.md:27]: I think it's worth adding a section that shows that you can pass your own session created by boto3. You don't need to use Dragoneye session factory

File: README.md [hunk 2]
```diff
@@ -62,10 +63,10 @@ azure_scan_output_directory = AzureScanner(token, azure_settings).scan()
 
 ### For collecting data from AWS
```
Comment (alonashkenazi) [inline README.md:64]: I'm assuming that you still need to work on the CLI documentation right?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # dragoneye
```
Comment (alonashkenazi) [inline README.md:1]: Things that need to be part of the readme:

- License
- How to triage a bug and how to report a bug
- How to open feature request (https://github.com/hashicorp/terraform/blob/main/BUGPROCESS.md)
- How to contribute (https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md)

**Review by alonashkenazi — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -21,34 +21,35 @@ pip install .
 
 ## Programmatic Usage
 Create an instance of one of the CollectRequest classes, such as AwsAccessKeyCollectRequest, AwsAssumeRoleCollectRequest, AzureCollectRequest and call the `collect` function. For example:
```
Comment (Tomerva) [inline README.md:23]: This text is outdated. The text should now note that each scanner requires the scan settings (AwsCloudScanSettinigs or AzureCloudScanSettings) and the either the session for aws, or the JWT bearer token.

File: README.md [hunk 5]
```diff
@@ -21,34 +21,35 @@ pip install .
...
     account_name='default', default_region='us-east-1', regions_filter=['us-east-1']
 )
 
-azure_settings = AzureCloudScanSettings(
-    commands_path='/Users/dev/python/dragoneye/azure_commands_example.yaml',
-    subscription_id='...',
-    account_name='my-account'
-)
+# Using environment variables
+session = AwsSessionFactory.get_session(profile_name=None, region='us-east-1')  # Raises exception if authentication is unsuccessful
```
Comment (Tomerva) [inline README.md:34]: Perhaps its worth noting that the region could also be taken from environment variable AWS_DEFAULT_REGION

File: README.md (hunk 3 above)
Comment (yi2020) [reply README.md:1]: @alonashkenazi please generate the CONTRIBUTING file in this repo and include in the same branch as the readme (CR-2005-create-readme-for-dragoneye) and I'll edit it.

Comment (yi2020) [reply README.md:1]: To clarify - you write the process in the CONTRIBUTING file, I just edit it.

File: README.md (hunk 5 above)
Comment (yi2020) [reply README.md:34]: I don't want to make the documentation it too complicated.

**Comment by yi2020** [comment]
Replacing this PR with https://github.com/indeni/dragoneye/pull/8

**Pull request closed without merging**